README.md

```diff
@@ -197,3 +204,43 @@ version of it. Now, you can use it in your article.
 >In case you don't have access to uploading images in Wordpress, contact
 Wordpress Administrator.
 >Remember to ALWAYS optimize all your images! It impacts the score heavily.
+
+## Publishig post
```

Comment:
@artur-rs this procedure doesn't seem to use news-and-ideas repositor. We should have procedure starting with `git clone <this_repo>`